### PR TITLE
Added doc on ingesting data

### PR DESCRIPTION
**Pull request by theSaarco**
Currently I could find no reference to Spark service docs (not spark-op). Have a link pointing nowhere - if we don't plan to document it, then this should be removed.

### CODE REVIEW

File: docs/data-collect-prep/ingesting_data.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+For distributed processing of very large datasets, MLRun integrates with the Spark processing engine, and provides
+facilities for executing pySpark code using a Spark service (which can be deployed by the platform when running MLRun
+as part of an Iguazio system) or through submitting the processing task to Spark-operator. The following pages provide
+additional details and code-samples:
+
+1. [Spark service](???) - **do we have a page for this? Are we documenting it?**
+2. [Spark operator](../runtimes/spark-operator.html)
+
+In a similar manner, Dask can be used for parallel processing of the data. To read data as a Dask data-frame, use the
```
Comment (jillnogold) [inline docs/data-collect-prep/ingesting_data.md]: Dask DataFrame

File: docs/data-collect-prep/ingesting_data.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+(ingesting_data)=
+# Ingesting data
+
+MLRun provides a set of tools and capabilities to streamline the task of data ingestion and processing. For an 
+end-to-end framework for data processing, management and serving MLRun has the feature-store capabilities, which are
```
Comment (jillnogold) [inline docs/data-collect-prep/ingesting_data.md]: For an end-to-end framework for data processing, management, and serving, MLRun has the feature-store capabilities...

File: docs/data-collect-prep/ingesting_data.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+os.environ["AWS_ACCESS_KEY_ID"] = "<access key ID>"
+os.environ["AWS_SECRET_ACCESS_KEY"] = "<access key>"
+
+source_url = "s3://input-data/input_data.csv"
+
+input_data = mlrun.get_dataitem(source_url).as_df()
+```
+
+This code runs locally (for example, in Jupyter) and relies on environment variables to supply credentials for data 
+access. See [this page](../store/datastore.html) for more info on the available data-stores, accessing them locally and
```
Comment (jillnogold) [inline docs/data-collect-prep/ingesting_data.md]: See {store}`datastore` for more info...

File: docs/data-collect-prep/ingesting_data.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+```python
+cleaned_data_frame = ingest_data_run.artifact('cleaned_data').as_df()
+```
+
+Note that running the function remotely may require attaching storage to the function, as well as passing storage
+credentials through project secrets. See the following pages for more details:
+
+1. [Attach storage to functions](../runtimes/function-storage.html)
+2. [Working with secrets](../secrets.html)
```
Comment (jillnogold) [inline docs/data-collect-prep/ingesting_data.md]: - {ref}`Function_storage_auto_mount`
- {ref}`secrets`

File: docs/data-collect-prep/ingesting_data.md (hunk 3 above)
Comment (theSaarco) [reply docs/data-collect-prep/ingesting_data.md]: Not sure what you mean - do you want to replace "this page" (which is the text that the link will show) with "store"?

File: docs/data-collect-prep/ingesting_data.md (hunk 4 above)
Comment (theSaarco) [reply docs/data-collect-prep/ingesting_data.md]: Again I'm not sure - what is `function_storage_auto_mount`? I've written the name of the page to be shown in the href. Do you want this to be replaced with `function_storage_auto_mount`? This is what the user will see - 
[function_storage_auto_mount](../runtimes/function-storage.html)

Comment (jillnogold) [reply docs/data-collect-prep/ingesting_data.md]: It's a way of cross-referencing pages that Gilad wanted to start usinghttps://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#targets-and-cross-referencing

File: docs/data-collect-prep/ingesting_data.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+Once the data is imported from its source, it can be processed using any framework. MLRun natively supports working
+with Pandas DataFrames and converting from and to its `DataItem` object.
+
+For distributed processing of very large datasets, MLRun integrates with the Spark processing engine, and provides
+facilities for executing pySpark code using a Spark service (which can be deployed by the platform when running MLRun
+as part of an Iguazio system) or through submitting the processing task to Spark-operator. The following pages provide
+additional details and code-samples:
+
+1. [Spark service](???) - **do we have a page for this? Are we documenting it?**
+2. [Spark operator](../runtimes/spark-operator.html)
```
Comment (jillnogold) [inline docs/data-collect-prep/ingesting_data.md]: - {ref}`spark-operator`

Leave out Spark service for now, it's in the list of TODOs...

File: docs/data-collect-prep/ingesting_data.md (hunk 4 above)
Comment (theSaarco) [reply docs/data-collect-prep/ingesting_data.md]: oh, ok. Got that.

File: docs/data-collect-prep/ingesting_data.md (hunk 3 above)
Comment (theSaarco) [reply docs/data-collect-prep/ingesting_data.md]: Fixed

**Pull request merged**